airflow/providers/amazon/aws/operators/boto3.py

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Callable, Sequence
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+from airflow.models import BaseOperator
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class Boto3BaseOperator(BaseOperator):
+    """Base boto3 operator for interating with AWS via boto3"""
+
+    template_fields: Sequence[str] = ("boto3_kwargs",)
+
+    def __init__(
+        self,
+        *args,
+        aws_conn_id: str = "aws_conn_id",
```

Comment:
```suggestion
        aws_conn_id: str | None = AwsBaseHook.default_conn_name,
```

Connection ID could be empty (`None`) in this case AwsBaseHook would use default `boto3` credentials strategy.
Default AWS Connection ID is `aws_default`